core/primitives/src/views.rs

```diff
@@ -317,7 +317,8 @@ pub struct StatusResponse {
     pub protocol_version: u32,
     /// Latest protocol version that this client supports.
     pub latest_protocol_version: u32,
-    /// Address for RPC server.
+    /// Address for RPC server.  Empty if node does not have RPC endpoint
+    /// enabled.
```

Comment:
I think it is better to make it `Option<String>`

Reply:
The reason I didn’t turn it into option is the ‘These types should only change when we cannot avoid this.’ suggestion near the top of the page. As far as I understand `StatusResponse` is used in inter-process messaging so changing it would be a breaking change?

Reply:
@mina86 This particular one is not used for inter-process messaging (you can tell that by two factors: (1) there is no BorshSerialization implementation on it, (2) reviewing all the usages you can see that it is only used in JSON RPC, which serializes the responses into JSON, so in most of the cases, where RPC is enabled, there will be no structural changes, and in the case when RPC is disabled one cannot get that response in the first place :smile:

Reply:
Done.